src/app/pages/HomePage/Graph/Graph/graph-utils.ts

```diff
@@ -38,7 +38,11 @@ export abstract class GraphUtils {
           y: 1.1 * height,
         }
       case Layer.consensus:
-        return initialValue
+        return {
```

Comment:
Looks like this broke `zoom out` button on desktop.

Reply:
See fix in https://github.com/oasisprotocol/explorer/pull/1413